README.md

```diff
@@ -10,6 +10,8 @@ An Ansible Playbook Bundle (APB) is a new method for defining and distributing c
 
 Read more about the Ansible Service Broker and Ansible Playbook Bundles in this [introduction](docs/introduction.md).
 
+**NOTE**: this broker will be based on the [open-service-broker-sdk](https://github.com/openshift/open-service-broker-sdk) project in the future.
```

Comment:
When will we rebase to sdk? I thought it won't be for a few months. So I'd rather not put that statement in now. I want as many contributions we can get between now and then, if there are any. If I saw a statement like that, I'd not want to contribute until the project was on SDK and stable.